mediacapture-streams/MediaStream-audio-only.https.html

```diff
@@ -15,12 +15,12 @@ <h1 class="instructions">Description</h1>
 <div id='log'></div>
 <script src=/resources/testharness.js></script>
 <script src=/resources/testharnessreport.js></script>
-<script src="/common/vendor-prefix.js" data-prefixed-objects='[{"ancestors":["navigator"], "name":"getUserMedia"}, {"ancestors":["window"], "name":"MediaStream"}]'></script>
+<script src="/common/vendor-prefix.js" data-prefixed-objects='[{"ancestors":["window"], "name":"MediaStream"}]'></script>
```

Comment:
Why not remove this vendor?

Reply:
I did a search for getUserMedia and removed those, but sure, I can remove all the MediaStream vendor prefixes too.